includes/html/print-alert-rules.php

```diff
@@ -200,9 +201,27 @@
             $status_msg = "All devices matching " . $rule['name'] . "  are OK";
         }
         if ((int) $sub['state'] === 1 || (int) $sub['state'] === 2) {
-            $ico   = 'exclamation';
-            $col   = 'danger';
-            $extra = 'danger';
+            switch ($severity) {
```

Comment:
This should probably be a generic function, so we use the same classes everywhere

Reply:
done :-)